core/api/tests/test_business_plans.py

```diff
@@ -578,6 +582,31 @@ def test_update_comment_draft_version(
         assert activities[0]["comment_secretariat"] == "Nu inchide telefonu"
         assert activities[0]["comment_types"] == [comment_type.id]
 
+    def test_is_updated(self, agency_user, _setup_bp_activity_create, business_plan):
+        self.client.force_authenticate(user=agency_user)
+
+        url = reverse("businessplan-list") + f"{business_plan.id}/"
+        data = {
+            "agency_id": business_plan.agency_id,
+            "year_start": business_plan.year_start,
+            "year_end": business_plan.year_end,
+            "status": "Agency Draft",
+            "activities": [_setup_bp_activity_create],
+        }
+        # update bp activity
+        response = self.client.put(url, data, format="json")
+        assert response.status_code == 200
+        new_id = response.data["id"]
+        activities = response.data["activities"]
+        assert activities[0]["is_updated"] is True
+
+        # update bp again without changes
```

Comment:
I think this test should contain the response from the get request. in this way you will have also the activity ids in the put request body. 
This approach is closer to the live requests